test/test_autograd.py

```diff
@@ -3127,6 +3127,38 @@ def test_set_data_preserve_pyobj(self):
         b.data = a
         self.assertTrue(b_id_saved == id(b))
 
+    def _run_and_capture_stderr(self, code):
+        import os
+        import subprocess
+
+        env = os.environ.copy()
+        env["PYTORCH_API_USAGE_STDERR"] = "1"
+        pipes = subprocess.Popen(
+            [sys.executable, '-c', code],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=env)
+        return pipes.communicate()[1].decode('ascii')
```

Comment:
Since this got copy-pasted from `test_logging.py`, maybe it should get moved to the common test code? That way, we don't have to copy paste. It would also be a good opportunity to name the function appropriately to mention that it's all about `PYTORCH_API_USAGE_STDERR`.